src/untangled/datomic/impl/components.clj

```diff
@@ -77,3 +79,7 @@
         (info "Deleting database" db-name url)
         (datomic/delete-database url)))
     (assoc this :connection nil)))
+
+(prefer-method io.aviso.exception/exception-dispatch clojure.lang.IPersistentMap clojure.lang.IDeref)
+(prefer-method print-method clojure.lang.IRecord clojure.lang.IDeref)
+(prefer-method clojure.pprint/simple-dispatch clojure.lang.IPersistentMap clojure.lang.IDeref)
```

Comment:
Are these potentially harmful? Could it crash if somehow those deps are not available? Specifically `io.aviso.exception`.